packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -1288,9 +1368,21 @@ export class Combobox
     const items: HTMLCalciteComboboxItemElement["el"][] = Array.from(
       this.el.querySelectorAll(ComboboxItemSelector),
     );
+
     return items.filter((item) => withDisabled || !item.disabled);
   }
 
+  private keyboardNavItems(): HTMLCalciteComboboxItemElement["el"][] {
```

Comment:
This could be a stateful property getter if `this.selectedAllComboboxItemReferenceEl` and `items` were also stateful properties.